pkg/cloudclient/aws/private.go

```diff
@@ -93,6 +93,35 @@ func newClient(ctx context.Context, logger ocmlog.Logger, accessID, accessSecret
 	return c, nil
 }
 
+func newClientFromProfile(ctx context.Context, logger ocmlog.Logger, profile, region, instanceType string, tags map[string]string) (*Client, error) {
+	cfg, err := config.LoadDefaultConfig(ctx,
+		config.WithSharedConfigProfile(profile),
+		config.WithRegion(region),
+	)
+
+	if err != nil {
+		return nil, err
+	}
+
+	c := &Client{
+		ec2Client:    ec2.NewFromConfig(cfg),
+		region:       region,
+		instanceType: instanceType,
+		tags:         tags,
+		logger:       logger,
+		output:       output.Output{},
+	}
+
+	// Validates the provided instance type will work with the verifier
+	// NOTE a "nitro" EC2 instance type is required to be used
+	if err := c.validateInstanceType(ctx); err != nil {
```

Comment:
Why does the function that creates the client also validate an instance?

Reply:
So i followed the newClient function right above this one ( line 65)

Reply:
https://github.com/openshift/osd-network-verifier/blob/705a76be3c678ce12b0802dd34dff0f06be4ece4/pkg/cloudclient/aws/private.go#L65

Reply:
`// Validates the provided instance type will work with the verifier`

Reply:
https://issues.redhat.com/browse/OSD-11483

We have a card to remove this